generate.go

```diff
@@ -21,9 +21,7 @@ import (
 var tmplFS embed.FS
 
 var shellFiles = []string{".gitignore", "shell.nix"}
-
-// TODO: we should also generate a .dockerignore file
-var buildFiles = []string{".gitignore", "development.nix", "runtime.nix", "Dockerfile"}
+var buildFiles = []string{".gitignore", "development.nix", "runtime.nix", "Dockerfile", "Dockerfile.dockerignore"}
```

Comment:
Does the name `Dockerfile.dockerignore` work? I thought it had to be named `.dockerignore`.

Reply:
See https://github.com/jetpack-io/devbox/pull/41#discussion_r958745018